selenium/codenvy-selenium-test/pom.xml

```diff
@@ -147,6 +155,11 @@
             <scope>runtime</scope>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.che.plugin</groupId>
+            <artifactId>che-plugin-github-server</artifactId>
+            <scope>runtime</scope>
```

Comment:
Yes, because there is implementation of **GitHubKey** DTO class in **che-plugin-github-server** which isn't referenced in tests directly.